commanderbot_ext/invite/invite_migrations.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+from commanderbot_lib.database.abc.versioned_file_database import VersionedFileDatabase
+
+
+async def m_1a_init_aliases_dates_hits(database: VersionedFileDatabase, data: dict):
```

Comment:
This doesn't look like it was changed from FAQ. That being said, I don't think it's needed in the first place because there haven't been any changes to the data format for `invites` yet. You can probably remove this file entirely.